source/engine/physics/NcPhysicsImpl.cpp

```diff
@@ -65,8 +68,8 @@ auto BuildPhysicsModule(const config::PhysicsSettings& settings, Registry* regis
 }
 #endif
 
-NcPhysicsImpl::NcPhysicsImpl(const config::PhysicsSettings& settings, Registry* registry, SystemEvents& events)
-    : m_pipeline{registry, settings.fixedUpdateInterval, events.rebuildStatics},
+NcPhysicsImpl::NcPhysicsImpl(const config::PhysicsSettings&, Registry* registry, SystemEvents& events)
+    : m_pipeline{registry, 1.0f / 60.0f, events.rebuildStatics},
```

Comment:
Is the plan to do away with fixedUpdateInterval permanently?

Reply:
Yes, `EngineSetting::timeStep` is a fixed time step for the whole update loop, making `fixedUpateInerval` redundant.